src/update-trigger-binding-behavior.js

```diff
@@ -29,13 +24,15 @@ export class UpdateTriggerBindingBehavior {
     targetObserver.originalHandler = binding.targetObserver.handler;
 
     // replace the element subscribe function with one that uses the correct events.
-    let handler = this.eventManager.createElementHandler(events);
+    let handler = new EventSubscriber(events);
     targetObserver.handler = handler;
   }
 
   unbind(binding, source) {
     // restore the state of the binding.
+    binding.targetObserver.handler.dispose();
     binding.targetObserver.handler = binding.targetObserver.originalHandler;
+    // console.log(binding.targetObserver.handler);
```

Comment:
Leftover from testing?

Reply:
I even had `fit(...` not cleaned up. Thanks!